app/models/dream.rb

```diff
@@ -33,51 +38,36 @@ def positivity
     end
   end
 
-  def get_sentiment
+  def dream_analysis
     alchemyapi = AlchemyAPI.new
-    response = alchemyapi.sentiment('text', self.text, { 'sentiment'=>1 })
-
-    if response['status'] == 'OK'
-      sentiment = response['docSentiment']['score']
+    keyword_response = alchemyapi.keywords('text', text, 'sentiment' => 1)
+    sentiment_response = alchemyapi.sentiment('text', text, { 'sentiment'=>1 })
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator `=>`.